scripts/prompter.js

```diff
@@ -0,0 +1,108 @@
+// }
+
+// OPD-4A
+// console.log([departmentInfo] + " is een leuke afdeling om te werken. Er werken op dit moment " + departments[departmentInfo].numberOfEmployees + " medewerkers.");
+
+//OPD-4B-C
+// const functionInfo = prompt("Je koos " + [departmentInfo] + ". Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in." +
+const functionInfo = prompt(`Je koos ${departmentInfo}. Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in.` +
+    "\n 0: " + departments[departmentInfo].jobs[0].title + "," +
+    "\n 1: " + departments[departmentInfo].jobs[1].title + "," +
+    "\n 2: " + departments[departmentInfo].jobs[2].title + "," +
+    "\n 3: " + departments[departmentInfo].jobs[3].title);
+
+switch (functionInfo) {
+    case "0":
+        // console.log("Je koos " + departments[departmentInfo].jobs[0].title + ". Een uitdagende rol! " + departments[departmentInfo].jobs[0].description);
+        document.getElementById("role-title").textContent = departments[departmentInfo].jobs[functionInfo].title;
+        document.getElementById("department-description").textContent = departments[departmentInfo].description;
+        document.getElementById("role-description").textContent = departments[departmentInfo].jobs[functionInfo].description;
+        break;
+    case "1":
+        // console.log("Je koos " + departments[departmentInfo].jobs[1].title + ". Een uitdagende rol! " + departments[departmentInfo].jobs[1].description);
+        document.getElementById("role-title").textContent = departments[departmentInfo].jobs[functionInfo].title;
+        document.getElementById("department-description").textContent = departments[departmentInfo].description;
+        document.getElementById("role-description").textContent = departments[departmentInfo].jobs[functionInfo].description;
+        break;
+    case "2":
+        // console.log("Je koos " + departments[departmentInfo].jobs[2].title + ". Een uitdagende rol! " + departments[departmentInfo].jobs[2].description);
+        document.getElementById("role-title").textContent = departments[departmentInfo].jobs[functionInfo].title;
+        document.getElementById("department-description").textContent = departments[departmentInfo].description;
+        document.getElementById("role-description").textContent = departments[departmentInfo].jobs[functionInfo].description;
+        break;
+    case "3":
+        // console.log("Je koos " + departments[departmentInfo].jobs[3].title + ". Een uitdagende rol! " + departments[departmentInfo].jobs[3].description);
+        document.getElementById("role-title").textContent = departments[departmentInfo].jobs[functionInfo].title;
+        document.getElementById("department-description").textContent = departments[departmentInfo].description;
+        document.getElementById("role-description").textContent = departments[departmentInfo].jobs[functionInfo].description;
+        break;
```

Comment:
Hier dus ook 😉 
```suggestion
    case "0":
    case "1":
    case "2":
    case "3":
        // console.log("Je koos " + departments[departmentInfo].jobs[functionInfo].title + ". Een uitdagende rol! " + departments[departmentInfo].jobs[functionInfo].description);
        document.getElementById("role-title").textContent = departments[departmentInfo].jobs[functionInfo].title;
        document.getElementById("department-description").textContent = departments[departmentInfo].description;
        document.getElementById("role-description").textContent = departments[departmentInfo].jobs[functionInfo].description;
        break;
```

Reply:
@Zwen1111 Thanks voor de feedback. Is een stuk schoner zo. Aangepast. Pull request vervolgens maar gemerged